webpack.config.js

```diff
@@ -1,8 +1,12 @@
+/* eslint-disable */
 const path = require('path');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 
 module.exports = {
   entry: './src/index.js',
```

Comment:
- [ ] **[OPTIONAL]** When I start your app I get this message. I would recommend you add a mode, like so:

  ```
  module.exports = {
    mode: 'development',
    entry: './src/index.js',
    ...
  ```

  ![rirow](https://user-images.githubusercontent.com/36542986/206357816-227fd2dc-6eb8-423b-ba10-071a165b32f4.jpg)